src/app/notifications/NotificationButton.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+
+interface Props {
+  onClick(): void;
+}
+
+/**
+ * an independent element fed into showNotification({body:
+ * attach your own functionality to its onClick when creating it.
+ * jsx children are the button's label
+ */
+export default class NotificationButton extends React.Component<Props> {
```

Comment:
For most components (especially super simple ones) function components are preferred over classes.

Reply:
good point, weird leftover